src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -1962,15 +1962,28 @@ export class PostgresStorageAdapter implements StorageAdapter {
 
     const wherePattern =
       where.pattern.length > 0 ? `WHERE ${where.pattern}` : '';
-    const qs = `SELECT count(*) FROM $1:name ${wherePattern}`;
-    return this._client
-      .one(qs, values, a => +a.count)
-      .catch(error => {
-        if (error.code !== PostgresRelationDoesNotExistError) {
-          throw error;
-        }
-        return 0;
-      });
+    let tempQs = '';
+
+    if (where.pattern.length > 0) {
+      tempQs = `SELECT count(*) FROM $1:name ${wherePattern}`;
+    } else {
+      tempQs = `SELECT reltuples AS approximate_row_count FROM pg_class WHERE relname = '` + className + "' LIMIT 1";
```

Comment:
Remove the templated string here and replace with 

`'SELECT reltuples AS approximate_row_count FROM pg_class WHERE relname = $1'`